bin/fin

```diff
@@ -107,6 +107,12 @@ DEFAULT_MACHINE_VBOX_RAM='1024' #mb
 DEFAULT_MACHINE_VBOX_HDD='50000' #mb
 DEFAULT_MACHINE_DO_SIZE='1gb' # digitalocean default size (512mb, 1gb, 2gb...)
 
+# Stats
+# fin sends a minimal ping with OS and fin version number
+DOCKSAL_STATS_TID='UA-93724315-1'
+DOCKSAL_STATS_URL='http://www.google-analytics.com/collect'
+DOCKSAL_STATS_OPTOUT=${DOCKSAL_STATS_OPTOUT:-0}
```

Comment:
We already have SKIP_VBOX_VERSION_CHECK, should we maybe keep alike options named consistently?
Like SKIP_STATS_GATHERING?